src/init.cpp

```diff
@@ -456,6 +456,7 @@ void SetupServerArgs(NodeContext& node)
     gArgs.AddArg("-proxy=<ip:port>", "Connect through SOCKS5 proxy, set -noproxy to disable (default: disabled)", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     gArgs.AddArg("-proxyrandomize", strprintf("Randomize credentials for every proxy connection. This enables Tor stream isolation (default: %u)", DEFAULT_PROXYRANDOMIZE), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     gArgs.AddArg("-seednode=<ip>", "Connect to a node to retrieve peer addresses, and disconnect. This option can be specified multiple times to connect to multiple nodes.", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-networkactive", "Enable all p2p network activity (default: 1). Could be changed by the `setnetworkactive' RPC command", ArgsManager::ALLOW_BOOL | ArgsManager::DEBUG_ONLY, OptionsCategory::CONNECTION);
```

Comment:
The open-quote character before `setnetworkactive` is a back-quote, but the close-quote character is a regular single-quote. There is one other place in this file that uses back-quote characters (addnode), but there they are balanced (back-quote for both open-quote and close-quote).

Reply:
https://www.gnu.org/prep/standards/html_node/Quote-Characters.html

TBH, I'm opposite to using back-quote characters for both opening and closing quotes. If you believe that quotes should be balanced, could they be `'`?

Reply:
ISTM the most common practice ATM is neither quotes nor backticks in the bitcoind help, e.g. ` bitcoind -help | grep "pruneblockchain\|getrawtransaction"`

Reply:
Also: s/Could/Can/

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19473#issuecomment-660453564).